tools/cross/format.py

```diff
@@ -117,7 +117,12 @@ def prettier(files: list[Path], check: bool = False) -> bool:
 
 
 def buildifier(files: list[Path], check: bool = False) -> bool:
-    cmd = [BUILDIFIER, "--mode=check" if check else "--mode=fix"]
```

Comment:
Bazel is annoyingly slow for simple tasks
I would personally prefer to use bazel to vendor the binary into a const path and then just invoke it directly without bazel.